### PR TITLE
Mesh 595/code clean up

### PR DESCRIPTION
**Pull request by JMoore96**

### CODE REVIEW

File: pallets/identity/rpc/runtime-api/src/lib.rs [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 use codec::Codec;
 use frame_support::dispatch::Vec;
+#[allow(unused_imports)]
```
Comment (adamdossa) [inline pallets/identity/rpc/runtime-api/src/lib.rs]: Can we just get rid of the imports if they aren't needed?

File: pallets/identity/rpc/src/lib.rs [hunk 2]
```diff
@@ -1,6 +1,7 @@
 use codec::Codec;
 use jsonrpc_core::{Error as RpcError, ErrorCode, Result};
 use jsonrpc_derive::rpc;
+#[allow(unused_imports)]
```
Comment (adamdossa) [inline pallets/identity/rpc/src/lib.rs]: Can we just remove?

File: pallets/identity/rpc/runtime-api/src/lib.rs (hunk 1 above)
Comment (JMoore96) [reply pallets/identity/rpc/runtime-api/src/lib.rs]: removed.

File: pallets/identity/rpc/src/lib.rs (hunk 2 above)
Comment (JMoore96) [reply pallets/identity/rpc/src/lib.rs]: removed as well.

**Pull request closed without merging**